backend/howtheyvote/scrapers/votes.py

```diff
@@ -912,3 +913,109 @@ def _extract_data(self, doc: BeautifulSoup) -> Fragment | None:
                 "geo_areas": geo_areas,
             },
         )
+
+
+class OEILSummaryIDScraper(BeautifulSoupScraper):
+    BASE_URL = "https://oeil.secure.europarl.europa.eu/oeil/en/procedure-file"
+
+    def __init__(
+        self,
+        vote_id: int,
+        day_of_vote: date,
+        reference: str,
+        request_cache: RequestCache | None = None,
+    ):
+        super().__init__(request_cache=request_cache)
+        self.day_of_vote = day_of_vote
+        self.vote_id = vote_id
+        self.reference = reference
+
+    def _url(self) -> str:
+        ref = f"?reference={self.reference}"
+        return f"{self.BASE_URL}{ref}"
+
+    def _extract_data(self, doc: BeautifulSoup) -> Fragment:
+        sections = doc.select(".es_product-section")
+        summary_id = None
+        for section in sections:
+            # TODO: Assuming it is the third section all the time, which is likely,
```

Comment:
@tillprochaska would you be fine with that assumption? We could still check that the heading matches and error out otherwise.